openedx/adg/admin_dashboard/applications/helpers.py

```diff
@@ -0,0 +1,103 @@
+"""
+Helpers for `admin_dashboard.applications` app.
+"""
+import json
+from datetime import datetime, timedelta
+
+from openedx.adg.lms.applications.models import UserApplication
+from openedx.adg.lms.applications.rules import is_adg_admin
+
+from .constants import WEEKLY_SUBMISSIONS_GRAPH_DATE_FORMAT
+
+
+def get_submitted_applications(user):
+    """
+    Returns all applications that have been submitted, for ADG admin, and for business unit admin, it returns only those
+    submitted applications that have been applied against the business unit that the user is admin of.
+
+    Arguments:
+        user (User): admin for whom submitted applications are to be fetched
+
+    Returns:
+        QuerySet: queryset containing submitted applications
+    """
+    submitted_applications = UserApplication.objects.submitted_applications().select_related_submission_dates()
+
+    if not (user.is_superuser or is_adg_admin(user)):
+        submitted_applications = submitted_applications.get_applications_for_business_unit_admin(
+            user.groups.all()
+        )
+
+    return submitted_applications
+
+
+def get_new_applications_count_and_percentage_change(submitted_applications):
+    """
+    Calculates the number of new applications in the current month and the percentage change between the number of
+    submitted applications in the current and previous month.
+
+    Arguments:
+        submitted_applications (QuerySet): queryset containing submitted applications
+
+    Returns:
+         new_applications_count (int): total number of submitted application in the current month
+         percentage (int): percentage change between submitted applications in the current and previous month
```

Comment:
This is actually a tuple. We should mention the return type as tuple and add this description against it.